python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/metadata.schema.yml

```
@@ -37,8 +37,11 @@ keys:
       hardware_requirements:
         type: dict
         description: |-
-          Defines the minimum hardware specifications required for the device.
-          Used for validation by the `anta_runner` role.
+          Defines hardware requirements for the device, validated by the `anta_runner` role.
+          For `min_power_supplies`, `min_fans`, `min_supervisors`, `min_line_cards`, and `min_fabric_cards`:
+          - If the value is a positive integer, `anta_runner` validates that the number of components is at least the specified minimum.
+          - If the value is 0, `anta_runner` validates that all populated components are in a healthy state.
+          - If not defined, the check for that component is skipped.
```

Let's have something more generic to avoid maintaining this long-term. Also, the definitions are inaccurate - 0 means we skip validation for that component.
```suggestion
          Defines hardware requirements for the device, validated by the `anta_runner` role.
          For the `min_*` keys:
          - Undefined (Default): Validate that all available slots are inserted.
          - Positive Integer: Validate that the number of components inserted is at least the specified minimum.
          - 0: Skip the validation for this specific component.
```

Also, for each `min_*` key below, let's update to something like: `Minimum number of power supplies required. Set to 0 to skip validation.`

---

Updated.